client/app/layout.tsx

```diff
@@ -5,16 +5,22 @@ import { Nav } from "./components/Nav";
 /* Instruments */
 import styles from "./styles/layout.module.css";
 import "./styles/globals.css";
+import Navigation from "./components/Layout/Navigation";
+import Footer from "./components/Layout/Footer";
+import footerLinks from "./data/dashboardLinks/footerLinks";
+import BCGovTypography from "./components/BCGovTypography";
 
 export default function RootLayout(props: React.PropsWithChildren) {
   return (
     <ReduxProvider>
       <html lang="en">
         <body>
           <section className={styles.container}>
-            <Nav />
+            <Navigation user="Fred Penner" />
+            <BCGovTypography />
```

Comment:
This was inconsistent on my machine--sometimes the Navigation component would show up with the proper formatting, sometimes not